jetty-util/src/main/java/org/eclipse/jetty/util/component/DumpableCollection.java

```diff
@@ -48,4 +55,18 @@ public void dump(Appendable out, String indent) throws IOException
         if (_collection!=null)
             ContainerLifeCycle.dump(out,indent,_collection);
     }
+    
+    public static class DumpableThread extends DumpableCollection
```

Comment:
This class is never used ? Not in this PR however, so it must not be in this PR.

Reply:
It is used by the updated dump in ManagedSelector

Reply:
Where ? I can't see it in this PR.

Reply:
Oh sorry, thought you were referring to DumpableCollection.  You are right about DumpableThread.